platform/windows/detect.py

```diff
@@ -347,12 +347,12 @@ def configure(env):
         env['ENV']['TMP'] = os.environ['TMP']
 
     # First figure out which compiler, version, and target arch we're using
-    if os.getenv("VCINSTALLDIR"):
+    if os.getenv("VCINSTALLDIR") and env.use_mingw == False:
```

Comment:
As per above comment, could simply be `!env["use_mingw"]`.

Reply:
Changed